test/data/htools/hail-alloc-secondary.json

```diff
@@ -294,5 +294,6 @@
   "request": {
     "name": "instance-to-be-converted-to-drbd",
     "type": "allocate-secondary"
-  }
+  },
+"enabled_hypervisors": ["xen-pvm"]
```

Comment:
I think this should also be indented by 2 spaces.

Reply:
Done.